test/e2e_node/mirror_pod_test.go

```diff
@@ -133,6 +134,60 @@ var _ = SIGDescribe("MirrorPod", func() {
 			err := deleteStaticPod(podPath, staticPodName, ns)
 			framework.ExpectNoError(err)
 
+			ginkgo.By("wait for the mirror pod to disappear")
+			gomega.Eventually(func() error {
+				return checkMirrorPodDisappear(f.ClientSet, mirrorPodName, ns)
+			}, 2*time.Minute, time.Second*4).Should(gomega.BeNil())
+		})
+	})
+	ginkgo.Context("when create a mirror pod without changes ", func() {
+		var ns, podPath, staticPodName, mirrorPodName string
+		ginkgo.BeforeEach(func() {
+		})
+		/*
+			Release: v1.23
+			Testname: Mirror Pod, recreate
+			Description: When a static pod's manifest is removed and readded, the mirror pod MUST successfully recreate. Create the static pod, verify it is running, remove its manifest and then add it back, and verify the static pod runs again.
+		*/
+		ginkgo.It("should successfully recreate when file is removed and recreated [NodeConformance]", func() {
```

Comment:
What happens if the config file is removed out and recreated multiple times in rapid succession?
Do we need to test that?

Reply:
I put a loop around the create/delete of the static pod locally and it works. We can followup with a separate PR to increase the test coverage.